### DISCUSSION TITLE
how to hide selected publications from the home page

### DISCUSSION THREAD

**Discussion by dua-arpit** [alshedivat/al-folio#2551] [Q&A] [answered]
Hi! Thank you for creating this; much appreciated. How can I hide selected publications from the About page? I tried looking for an option but could not find it

**Comment by george-gca** [comment, marked as answer]
It is the opposite, you have to explicitly add the publication as [selected={true}](https://github.com/alshedivat/al-folio/blob/369f0b74c9a412509b1b3f4a1a3b30e53fc9b65a/_bibliography/papers.bib#L60C3-L60C18) for it to appear in the about page.

**Reply by dua-arpit** [reply]
Sorry, I was looking for the variable selected_papers and found it in about.md. Thanks
